MFVNeutralino/test/TriggerStudies/filtercheck.py

```diff
@@ -39,10 +48,12 @@
         samples = Samples.HToSSTodddd_samples_2017
         #samples = Samples.all_signal_samples_2017
         #samples += Samples.leptonic_samples_2017
+        samples = [getattr(Samples, 'WplusHToSSTodddd_tau1mm_M55_2017')] 
```

Comment:
It's easy to introduce a bug by having samples defined in the top line and then overwritten in this bottom one. I think it's best to do one or the other!